src/librustdoc/html/render.rs

```diff
@@ -4461,6 +4448,13 @@ fn item_primitive(w: &mut fmt::Formatter, cx: &Context,
     render_assoc_items(w, cx, it, it.def_id, AssocItemRender::All)
 }
 
+fn item_keyword(w: &mut fmt::Formatter, cx: &Context,
+                it: &clean::Item,
+                _p: &str) -> fmt::Result {
+    document(w, cx, it)?;
+    render_assoc_items(w, cx, it, it.def_id, AssocItemRender::All)
```

Comment:
Does this need to have associated items? I don't expect us to link keywords to anything.

Reply:
Just like primitives. :)

Reply:
Actually, we link loads of things to primitives, because function signatures will have links to them. Keywords, on the other hand, don't have the same association. Since they're not *types*, per se, there's nothing that will automatically want to reference them. (Unless we want to link every function's `fn` keyword to that page, every `impl` block to that page, the `trait`/`struct`/`enum`/etc in type/trait definitions to those pages, etc... I don't quite think it will be useful enough to edge out over the noise of having yet another link.)

Reply:
True, I'll remove it then.